docs/en/latest/plugins/referer-restriction.md

```diff
@@ -2,7 +2,7 @@
 title: referer-restriction
 keywords:
   - APISIX
-  - Plugin
+  - API Gateway
   - Referer restriction
   - referer-restriction
 description: This document contains information about the Apache APISIX referer-restriction Plugin.
```

Comment:
Please refer to the Chinese modification, keep the characters between 130 and 150.